flexmeasures/cli/data_add.py

```diff
@@ -1537,7 +1115,6 @@ def add_schedule_for_storage(  # noqa C901
     "power_sensor",
     type=SensorIdField(),
     required=True,
-    cls=DeprecatedOption,
     deprecated=["--sensor-id"],
```

Comment:
These two lines should go, as when I run `± flexmeasures add schedule --help`, I get: 

`TypeError: Parameter.__init__() got an unexpected keyword argument 'deprecated'`